model/project_ref.go

```diff
@@ -1658,27 +1658,18 @@ func FindOneProjectRefWithCommitQueueByOwnerRepoAndBranch(owner, repo, branch st
 
 // EnableWebhooks returns true if a hook for the given owner/repo exists or was inserted.
 func EnableWebhooks(ctx context.Context, projectRef *ProjectRef) (bool, error) {
-	hook, err := FindGithubHook(projectRef.Owner, projectRef.Repo)
-	if err != nil {
-		return false, errors.Wrapf(err, "finding GitHub hook for project '%s'", projectRef.Id)
-	}
-	if hook != nil {
-		projectRef.TracksPushEvents = utility.TruePtr()
-		return true, nil
-	}
-
 	settings, err := evergreen.GetConfig(ctx)
 	if err != nil {
 		return false, errors.Wrap(err, "finding evergreen settings")
 	}
 
-	hook, err = SetupNewGithubHook(ctx, *settings, projectRef.Owner, projectRef.Repo)
-	if err != nil {
+	hasApp, err := settings.HasGitHubApp(ctx, projectRef.Owner, projectRef.Repo, nil)
+	if err != nil || !hasApp {
 		// don't return error:
 		// sometimes people change a project to track a personal
 		// branch we don't have access to
 		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
This isn't going to log in the !hasApp case, because grip.Error will noop if err is nil

Reply:
updated